.github/workflows/ci-matrix-build.yml

```
@@ -0,0 +1,167 @@
+          persist-credentials: false
+          ref: ${{ inputs.matrix_branch }}
+          path: .private
+      - name: Build workflow
+        id: build-workflow
+        uses: ./.github/actions/workflow-build
+        with:
+          workflows: ${{ inputs.workflows }}
+          # When sideloading from another branch, the matrix checkout is prefixed with
+          # '.private'. Otherwise use the path as-is from the current checkout
+          # (e.g. 'ci/matrix.yaml' for the default callers).
+          matrix_file: ${{ inputs.matrix_branch != '' && ".private/ci/matrix.yaml" || "ci/matrix.yaml" }}
```

actionlint (1.7.12)

[error] got unexpected character '"' while lexing expression, expecting 'a'..'z', 'A'..'Z', '_', '0'..'9', ''', '}', '(', ')', '[', ']', '.', '!', '<', '>', '=', '&', '|', '*', ',', ' '. do you mean string literals? only single quotes are available for string delimiter

(expression)